tests/network/openvpn_client.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test OpenVPN on two machines. This one is client.
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base 'consoletest';
+use testapi;
+use lockapi;
+use mm_network;
+use utils qw(systemctl zypper_call exec_and_insert_password);
+use strict;
+
+sub run {
+    my ($self) = @_;
+    select_console "root-console";
+
+    # Configure static network, disable firewall
+    configure_default_gateway;
+    systemctl 'stop ' . $self->firewall;
+    configure_static_ip('10.0.2.102/24');
+    configure_static_dns(get_host_resolv_conf());
+
+    # Install openvpn
+    zypper_call('in openvpn');
+    assert_script_run('cd /etc/openvpn');
```

Comment:
not clear why you doing this while most part of code using absolute paths anyway. Or remove this line or drop '/etc/openvpn' from code below to really using the fact that we are in "right" directory

Reply:
I do that for the 'diversity' to cover both relative and absolute paths in the config :smile: Should I change that?

Reply:
no if you doing this in purpose